src/pages/Courses/sections/StudyPath/StudyPath.tsx

```diff
@@ -0,0 +1,27 @@
+import { Paragraph, Title } from '../../../../components';
+import StageCard from '../../../../components/StageCard/StageCard';
+import { pathCoursesList } from '../coursesData';
+
+import './StudyPath.scss';
+
+export const StudyPath = () => {
+  return (
+    <section className="study-path container">
+      <div className="study-path content">
+        <Title text="Choose what you want to learn" hasAsterisk />
+        <Paragraph>
+          A full-stack developer is someone who has expertise in both frontend (what users see) and
+          backend (server and database) development. This dual skill set enables them to supervise
+          and implement projects from start to finish. Businesses today prioritize hiring full-stack
+          developers because they can efficiently bridge various technological aspects, resulting in
+          faster product development.
+        </Paragraph>
+        <div className="stages">
+          {pathCoursesList.map((course) => (
+            <StageCard key={course.id} {...course} />
```

Comment:
obviously it's easier to spread those props, but in order to find out what those props are, you have to look it up in the `StageCard` component. It would be better to have those props fully written

Reply:
got it